pyproject.toml

```
@@ -24,10 +24,10 @@ dependencies = [
   "cvprac~=1.3.1",
   "pydantic>=2.1.1,<2.7.0",
   "pydantic-extra-types>=2.1.0",
+  "eval-type-backport~=0.1.3",  # Support newer typing features in older Python versions (required until Python 3.9 included)
```

This does not mean what you think it does
```suggestion
  "eval-type-backport~=0.1.3",  # Support newer typing features in older Python versions (required until Python 3.9 support is removed)
```